modules/EvseV2G/iso_server.cpp

```diff
@@ -2435,10 +2366,10 @@ enum v2g_event iso_handle_request(v2g_connection* conn) {
 
         /* Configure session id */
         memcpy(exi_out->V2G_Message.Header.SessionID.bytes, &conn->ctx->evse_v2g_data.session_id,
-               iso1MessageHeaderType_SessionID_BYTES_SIZE);
+               iso2_sessionIDType_BYTES_SIZE);
 
-        /* We always set bytesLen to iso1MessageHeaderType_SessionID_BYTES_SIZE */
-        exi_out->V2G_Message.Header.SessionID.bytesLen = iso1MessageHeaderType_SessionID_BYTES_SIZE;
+        /* We always set bytesLen to iso2_MessageHeaderType_SessionID_BYTES_SIZE */
+        exi_out->V2G_Message.Header.SessionID.bytesLen = iso2_sessionIDType_BYTES_SIZE - 1;
```

Comment:
-1 causes a Failed_UnknowSession, as the byte length is set to 7. And the car sends a false SessionID with only 7 bytes. Therefore delete -1.

Reply:
Good catch.
Fixed. That creapt in because I wasn't sure about the null terminator initially. Removed it for the chars, but I must have added it to the bytes accidentally.